generated/user_existence/post_user_existence.json

```diff
@@ -0,0 +1,5 @@
+{
+  "person_demographics": {
+    "ssn": "121233211"
```

Comment:
If you don't enter an SSN, it takes you to the _Verify Identification_ page (with a list of questions).